client/utils/utils.c

```diff
@@ -431,3 +431,20 @@ void time_to_string(time_t time, char* buf)
 	localtime_r(&time, &tm);
 	strftime(buf, 50, "%Y-%m-%d %H:%M:%S", &tm);
 }
+
+void replace_all_control_characters(char *string, int length, char symbol)
+{
+	static char *control_chars = "\n\t\v\b\r\f\a ";
+
+	int control_chars_len = strlen(control_chars);
+	int index = 0;
+	while(string[index] != 0 && (length < 0 || index < length)) {
```

Comment:
It can be easier to use ascii code value to detect them. Like 
value < 32 || value > 126. https://en.wikipedia.org/wiki/ASCII

Reply:
Makes sense. I will do. Space (32) will be excluded too.

Reply:
Updated